docs/src/whatsnew/latest.rst

```diff
@@ -53,6 +53,10 @@ This document explains the changes made to Iris for this release
    enable lazy computation of rotated wind vector components (:issue:`4934`,
    :pull:`4972`)
 
+#. `@pp-mo`_ supported delayed saving of lazy data, when writing to the netCDF file
+   format.  See : :ref:`delayed netCDF saves <delayed_netcdf_save>`.
```

Comment:
```suggestion
   format.  See :ref:`delayed netCDF saves <delayed_netcdf_save>`.
```
Has something gone wrong with your formatting here?

Reply:
Not exactly, I meant "See : <link>", but the extra colon is not really  needed -- will remove.